meson_options.txt

```diff
@@ -6,6 +6,8 @@ option('rootlibdir', type : 'string',
        description : '''[/usr]/lib/x86_64-linux-gnu or such''')
 option('rootprefix', type : 'string',
        description : '''override the root prefix''')
+option('link-udev-shared', type : 'boolean',
+       description : 'link systemd-udev and its helpers to libsystemd-shared.so')
```

Comment:
install_rpath for systemd-udevd and udevadm should be set to ```rpath```, otherwise we embed an unnecessery RUNPATH for the static build:
```
$ objdump -x bin/udevadm | grep -E "(PATH|shared)"
  RUNPATH              /lib/systemd
```